apps/explorer/src/pages/transaction-result/programmable-transaction-view/Commands.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { type ProgrammableTransactionCommand } from '@mysten/sui.js';
+
+import { Command } from '~/pages/transaction-result/programmable-transaction-view/Command';
+import { TableHeader } from '~/ui/TableHeader';
+
+interface Props {
+    commands: ProgrammableTransactionCommand[];
+}
+
+export function Commands({ commands }: Props) {
+    if (!commands?.length) {
+        return null;
+    }
+
+    return (
+        <div data-testid="programmable-transactions-commands" className="mt-8">
+            <section className="pt-4 pb-4">
```

Comment:
Would also _probably_ remove this spacing and just depend on composing component to provide it